cpp/src/Slice/Parser.cpp

```diff
@@ -639,14 +638,14 @@ namespace
         }
     }
 
-    StringList splitComment(const string& c, bool stripMarkup)
+    StringList splitComment(string_view c, function<string(string)> linkFormatter, bool stripMarkup)
     {
-        string comment = c;
+        string comment = string{c};
+        string::size_type pos = 0;
```

Comment:
Sure, I switched it to take a `string`, since it's a simpler type.
But, this helper function is only called from one place, which will never use move semantics.